scripts/e2e-test-local-cluster-config.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
1. Consider renaming this to `scripts/e2e-test-local-cluster-config.json.SAMPLE` or similar, so other callers don't use it by accident and connect to a non-existent database. It would then be a local tester's responsibility to copy the sample file to the correct name.
2. If you haven't already got a task for it, please update one of the READMEs in this repo and our runbook with instructions for local testing. :)